domain-models-runtime/src/test/java/org/folio/DemoRamlRestTest.java

```diff
@@ -344,6 +345,38 @@ public void invalidCqlClosesConnection(TestContext context) {
     }
   }
 
+  /**
+   * Check that all Set-Cookie entries are returned.
+   *
+   * <p>See {@link org.folio.rest.impl.BooksDemoAPI#postRmbtestsBooks}
+   * @throws JsonProcessingException
+   */
+  @Test
+  public void testMultiHeaders() throws JsonProcessingException {
+    Data d = new Data();
+    d.setAuthor("a");
+    d.setGenre("g");
+    d.setDescription("description");
+    d.setTitle("title");
+    d.setDatetime(new Datetime());
+    d.setLink("link");
+
+    Book b = new Book();
+    b.setData(d);
+    b.setStatus(0);
+    b.setSuccess(true);
+
+    ObjectMapper om = new ObjectMapper();
+    String book = om.writerWithDefaultPrettyPrinter().writeValueAsString(b);
+    given()
+        .spec(tenant).header("Content-Type", "application/json")
+        .body(book).post("/rmbtests/books")
+        .then()
+        .statusCode(201)
+        .cookie("a", RestAssuredMatchers.detailedCookie().value("1"))
+        .cookie("b", RestAssuredMatchers.detailedCookie().value("2"));
```

Comment:
Think you can just do `.cookie("a", "1")`. `detailedCooke` is for when you have other properties like `HttpOnly` etc.

The format for a detailed cookie is `a=1; HttpOnly; Max-Age=123`. I don't think we have to test for all the possible values. But just to be on the safe side, you could make one of the cookies be detailed and then test for them with the `detailedCooke`.

Reply:
added more beefy cookie example